atomic_reactor/util.py

```diff
@@ -445,3 +451,21 @@ def get_exported_image_metadata(path):
     metadata['sha256sum'] = s.hexdigest()
     logger.debug('sha256sum: %s', metadata['sha256sum'])
     return metadata
+
+
+def print_version_of_tools():
```

Comment:
This information will also be useful to the `koji_promote` plugin. How about renaming this function to something like `get_version_of_tools()`, returning a dict?

Then calls to `print_version_of_tools()` can be replaced with `for tool, version in versions.items(): logger.info(...)`.